library/src/main/java/com/github/amlcurran/showcaseview/ShowcaseView.java

```diff
@@ -658,6 +767,14 @@ public Builder useDecorViewAsParent() {
             this.parentIndex = -1;
             return this;
         }
+        /**
+         * Call this first if you plan to use other Builder methods
+         * using the second button
+         */
+        public Builder addSecondButton(){
```

Comment:
so I think a better way of doing this would be to only have one method `public Builder addButtonTwo(int buttonResourceId)`. rather than having two buttons inflated as it currently does